client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -184,14 +198,24 @@ class Upload extends React.Component {
 				</span>
 				<img src={ theme.screenshot } />
 				<span className="theme-upload__description">{ theme.description }</span>
+				<Button href={ tryandcustomize.getUrl( theme ) }>
+					{ tryandcustomize.label }
+				</Button>
+				<Button primary onClick={ this.onActivateClick }>
+					{ activate.label }
+				</Button>
 			</div>
 		);
 	}
 
 	render() {
-		const { translate, inProgress, complete, failed } = this.props;
+		const { translate, inProgress, complete, failed, siteId, selectedSite } = this.props;
 		return (
 			<Main>
+				<QueryActiveTheme siteId={ siteId } />
+				<ThanksModal
+					site={ selectedSite }
+					source={ 'details' } />
```

Comment:
With a string value like this, can be shortened to `source="details"`. But maybe change `source` to `upload`?